buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkIcons.kt

```diff
@@ -60,9 +59,6 @@ data class BpkIcon(
               }
               .filterValues { it != null }
               .let { it as Map<Type, String> },
-            autoMirror = files
-              .filter { it.nameWithoutExtension.startsWith(name) }
-              .any { it.readText().contains("android:automirrored=\"true\"", ignoreCase = true) },
```

Comment:
the icons are already automirrored - we removed the logic to auto mirror them again based on this prop (in the past auto mirroring wasn't applied correctly in compose) - this removes the generation part as well